templates/tfengine/recipes/resources.hcl

```diff
@@ -1463,6 +1463,77 @@ schema = {
         }
       }
     }
+    kubernetes_service_accounts = {
+      description = "Kubernetes service accounts. See <https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/service_account>."
+      type        = "array"
+      items = {
+        additionalProperties = false
+        required = [
+          "name",
+          "namespace",
+          "google_service_account_email",
+          "provider"
+        ]
+        properties = {
+          name = {
+            description = "Name of the KSA."
+            type        = "string"
+          }
+          namespace = {
+            description = "Namespace to where the KSA will be created."
+            type        = "string"
+          }
+          google_service_account_email = {
+            description = "Email of the google service account the kubernetes service account should use to authenticate with other resources."
+          }
+          provider = {
+            description = "Kubernetes provider."
```

Comment:
Is this necessary? I think it's only needed when there are multiple clusters in the same deployment, and we would like to distinguish which is which and create kubernetes resources accordingly. Seems not needed to be in `required` list.

And the description needs to be a bit more detailed, i.e. it's the alias of the kubernetes provider used to authenticate to the cluster, so that the ksa can be created for the right cluster... please reword but something like that.

Reply:
Done.